assets/dev/js/editor/elements/views/base.js

```diff
@@ -582,10 +582,14 @@ BaseElementView = BaseContainer.extend( {
 	},
 
 	renderUI: function() {
-		this.renderStyles();
 		this.renderCustomClasses();
 		this.renderCustomElementID();
 		this.enqueueFonts();
+
+		_.defer( () => {
```

Comment:
@matipojo 

Maybe it'll be better to find the root cause for the global colors' unavailability?
Or do you think it's fine for now?

Reply:
I think it's OK for now